tests/generation/test_utils.py

```diff
@@ -780,7 +780,7 @@ def test_sample_generate(self):
                 forced_eos_token_id=model.config.forced_eos_token_id,
                 max_length=max_length,
             )
-            logits_warper_kwargs, logits_warper = self._get_warper_and_kwargs(num_beams=1)
+            logits_warper_kwargs, logits_warper = self._get_warper_and_kwargs(num_beams=2)
```

Comment:
This test is not beam sampling test, but I just give it 2 in order it can get `min_tokens_to_keep = 2`.